Assets/Scripts/Tools/Editor/DeveloperControls.cs

```diff
@@ -0,0 +1,242 @@
+            scrollPos.y = 0;
+
+            //---------------------
+            // EXECUTE THE COMMANDS
+            //---------------------
+            if (inputModified[0] == "help") // checks for help command which shows documentation
+            {
+                AddToConsoleLog("Documentation: https://docs.google.com/document/d/1v9Pv3NSl3cyLKnmF_MCTEz9_mHWrwQd4CHd3luqehxo/edit?usp=sharing");
+            }
+            else if (inputModified[0] == "killable") // checks for setkillable command
+            {
+                try // to catch missing or incorrect parameters or values
+                {
+                    if (inputModified[1] == "true")
+                    {
+                        SetKillable(true); // sets killable to true
+                    }
+                    else if (inputModified[1] == "false")
+                    {
+                        SetKillable(false); // sets killable to false
+                    } else
+                    {
+                        throw new Exception();
+                    }
+                }
+                catch (Exception)
+                {
+                    AddToConsoleLog("Error: missing true or false"); // tells the user if there is an error
+                }
+            }
+            else if (inputModified[0] == "kill") // checks for kill player command
+            {
+                KillPlayer();
+            }
+            else if (inputModified[0] == "restart") // checks for restart command
+            {
+                Restart();
+            }
+            else if (inputModified[0] == "spawn") // checks for spawn enemy command
+            {
+                try
+                {
+                    SpawnEnemy(inputModified[1]); // checks for exceptions, whether it is missing input or invalid enemy names
+                }
+                catch (IndexOutOfRangeException)
+                {
+                    AddToConsoleLog("Error: must enter an enemy");
+                }
+                catch (Exception)
+                {
+                    AddToConsoleLog("Error: enemy does not exist");
+                }
+            }
+
+        }
+    }
+
+    bool CheckCommand(string[] command) // checks if the command inputted exists
+    {
+        if (commandArray.Contains<string>(command[0]))
+        {
+            return true;
+        }
+        else
+        {
+            AddToConsoleLog("Error: Command \"" + userInput + "\" doesn't exist!");
+            return false;
+        }
+    }
+
+    //-------------------------------------
+    // INDIVIDUAL METHODS FOR ALL COMMANDS
+    //-------------------------------------
+
+    void SetKillable(bool state) // sets the player state to killable or unkillable
+    {
+        player = GameObject.Find("Player"); // Finds the player in the scene
+
+        if (state) 
+        {
+            player.GetComponent<PlayerKillable>().isKillable = true; // sets killable to true
```

Comment:
Make sure you pull from the develop branch and test that your code works with the existing code